### ISSUE TITLE
[4.0] Unable to sort articles list by clicking column header

### ISSUE THREAD

**Issue by PhilETaylor** [joomla/joomla-cms#18629]
### Steps to reproduce the issue

Go to the Articles list.
Click the ID column header

### Expected result

list of articles reorders by id

### Actual result

nothing happens - except more flying focus JS errors - might be related might not be... 
You can still use the table options to set the ordering by another column though

**Comment by brianteeman** [comment]
Confirmed
```
Uncaught TypeError: Cannot read property 'value' of null
    at t.checkFilter (searchtools.min.js:1)
    at searchtools.min.js:1
    at Array.forEach (<anonymous>)
    at t.init (searchtools.min.js:1)
    at new t (searchtools.min.js:1)
    at HTMLDocument.<anonymous> (searchtools.min.js:1)
```

**Comment by ciar4n** [comment]
Duplicate https://github.com/joomla/joomla-cms/issues/18242

**Comment by ghost** [comment]
closed as duplicate Report #18242. 

Thanks for Hint @ciar4n
